guardrail.ini

```diff
@@ -1,14 +1,14 @@
 [folder]
-# Comma seperated source folders if more than one directory
-source_folder = C:\public_repo\python_guardrails\guardrails
-# Comma seperated test folders if more than one directory
-test_folder = C:\public_repo\python_guardrails\test
-pytest_root = C:\public_repo\python_guardrails
-report_folder = C:\public_repo\python_guardrails\guardrail_report
+# space seperated source folders if more than one directory
+source_folder = C:\Projects\python_guardrails\guardrails C:\Projects\python_guardrails\test_resource
+# space seperated test folders if more than one directory
+test_folder = C:\Projects\python_guardrails\test
+pytest_root =
+report_folder = C:\Projects\python_guardrails\guardrail_report
 [python]
 python = python
 # path to the .pylintrc file if specific linting or leave empty after =
-pylint_rc_file = C:\public_repo\python_guardrails\.pylintrc
+pylint_rc_file = C:\Projects\python_guardrails\.pylintrc
```

Comment:
Should we be using OS specific paths here?

Reply:
this file is not anymore used